src/main/kotlin/no/nav/familie/ef/sak/vilkår/dto/TidligereVedtaksperioderDto.kt

```diff
@@ -49,12 +56,17 @@ fun TidligereInnvilgetVedtak.tilDto() =
         harTidligereBarnetilsyn = this.harTidligereBarnetilsyn,
         harTidligereSkolepenger = this.harTidligereSkolepenger,
         periodeHistorikkOvergangsstønad = this.periodeHistorikkOvergangsstønad.tilDto(),
+        periodeHistorikkBarnetilsyn = this.periodeHistorikkBarnetilsyn.tilDto(),
     )
 
 private fun List<GrunnlagsdataPeriodeHistorikkOvergangsstønad>.tilDto() = this.map { it.tilDto() }
     .slåSammenPåfølgendePerioderMedLikPeriodetype()
     .sortedByDescending { it.fom }
 
+@JvmName("tilDtoBarnetilsyn")
```

Comment:
Kan vi heller kalle denne `tilDtoBarnetilsyn` (og den andre `tilDtoOvergangsstønad`)?
Vil være mer "entydig" for senere